addons/l10n_cz/data/tax_report.xml

```diff
@@ -0,0 +1,831 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
wondering why we use single quote everywhere here but i wont block the pr for that